lib/widgets/home.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:mixeditems/providers/provider.dart';
+import 'package:provider/provider.dart';
+
+class Home extends StatelessWidget {
```

Comment:
I would suffix Widgets with a "...Widget" to make things clear

Reply:
Oh Okay, I changed others classes names which extends StatefulWidget or StatelessWidget.